sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -329,31 +343,32 @@ public void UpsertCdsSpreadCurves(string scope, DateTimeOffset effectiveAt, stri
                         },
                         definition: instrument));
         }
-        
+
         /// <summary>
         /// This method add instruments to a portfolio (specified by its scope and code),
         /// by creating a transaction request with the instrument's LUID.
         /// </summary>
-        private void AddInstrumentsTransactionToPortfolio(
+        public void AddInstrumentsTransactionToPortfolio(
             List<string> luids,
-            string portfolioScope,
-            string portfolioCode,
+            string scope,
+            string portfolioID,
```

Comment:
the LUSID API generally uses `code` is there are reason this has been changed to `ID`?

Reply:
No - there was a mix of ID and code scattered around the place and I tried to correct after I realised I was using the wrong one. Must have missed this one.